config/toml.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+const DefaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Base Configuration                            ###
+###############################################################################
+
+log-level = "{{ .BaseConfig.LogLevel }}"
+oracle-mnemonic = "{{ .BaseConfig.OracleMnemonic }}"
+data-encryption-key-file = "{{ .BaseConfig.DataEncryptionKeyFile }}"
+
+###############################################################################
+###                         Panacea Configuration                           ###
+###############################################################################
+
+[panacea]
+
+grpc-addr = "{{ .Panacea.GRPCAddr }}"
```

Comment:
I guess we need the `rpc-addr` (26657 stuff) instead of this now, especially for subscribing Tendermint events..

Reply:
It seems that we need to communicate with grpc to get the proof information of the query.

Reply:
Ok. I think we need gRPC connection to broadcast Txs for voting stuffs as well.